examples/amp-ad/sticky.html

```diff
@@ -71,7 +71,7 @@ <h1>
       height="1"
       id="i-amphtml-demo-id"
       type="fake-delayed"
-      sticky="bottom"
+      sticky
```

Comment:
I think this change can be removed from the PR.

Reply:
I just made a commit where I put that part back, tbh I'm not sure why it was even removed (maybe during a git pull upstream command)
_

Reply:
No worries!